lib/Runtime/Library/TypedArray.cpp

```diff
@@ -46,6 +46,54 @@ namespace Js
     {
     }
 
+    inline JsUtil::List<Var, ArenaAllocator>* IterableToList(RecyclableObject *object, ScriptContext *scriptContext, ArenaAllocator *tempAlloc)
```

Comment:
Nit: calling a param `tempAlloc `does not make sense in the context for this function. For this function this is just an allocation. Would it good just to call it `alloc`? what do you think?